### ISSUE TITLE
Latest release v2.1.4 doesn't include any "dist"

### ISSUE THREAD

**Issue by john-dupuy** [greatSumini/react-facebook-login#74]
@greatSumini I ran the following:
```console
$ yarn add @greatsumini/react-facebook-login@v2.1.3
$ ls node_modules/@greatsumini/react-facebook-login 
CHANGELOG.md  dist  LICENSE  package.json  README.md  renovate.json
```
And then updated to the latest v2.1.4
```console
$ yarn add @greatsumini/react-facebook-login@v2.1.4
$ ls node_modules/@greatsumini/react-facebook-login
LICENSE  package.json  README.md  renovate.json
```
And no `dist` is included.

When importing the component like 
```javascript
import FacebookLogin from '@greatsumini/react-facebook-login';
```
There is an error 
```console
./src/login.js
Cannot find module: '@greatsumini/react-facebook-login'. Make sure this package is installed.

You can install this package by running: yarn add @greatsumini/react-facebook-login.
```

**Comment by john-dupuy** [comment]
I think this might be due to the changes in `prepublish`? 

https://github.com/greatSumini/react-facebook-login/commit/ffbb0a473f28b38c4fd5b1bd2b11bdd85c16efd8

**Comment by greatSumini** [comment]
OMG 😱 thank u for report!
I'll handle it right away.

**Comment by greatSumini** [comment]
@john-dupuy 
Fixed in v2.1.5
thank you very much 🙇🏻

**Comment by john-dupuy** [comment]
Thanks so much for the quick fix @greatSumini! :1st_place_medal:
